CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+IPDK uses GitHub to manage reviews of pull requests.
+
+* If you are a new contributor see: [Steps to Contribute](#steps-to-contribute)
+
+* If you have a trivial fix or improvement, go ahead and create a pull request,
+  addressing (with `@...`) a suitable maintainer of this repository (see
+  [MAINTAINERS.md](MAINTAINERS.md)) in the description of the pull request.
+
+* Be sure to sign off on the [DCO](https://github.com/probot/dco#how-it-works).
+
+## Steps to Contribute
+
+Should you wish to work on an issue, please claim it first by commenting on the
+GitHub issue that you want to work on it. This is to prevent duplicated efforts
+from contributors on the same issue.
+
+## Pull Request Checklist
+
+* Branch from the main branch and, if needed, rebase to the current main branch
+  before submitting your pull request. If it doesn't merge cleanly with main
+  you may be asked to rebase your changes.
+
+* Commits should be as small as possible, while ensuring that each commit is
+  correct independently (i.e., each commit should compile and pass tests).
```

Comment:
Can we elaborate on what kind of test we expect to pass?

Reply:
Will do!